openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -88,6 +88,7 @@ def to_representation(self, user):
             "level_of_education": AccountLegacyProfileSerializer.convert_empty_to_None(profile.level_of_education),
             "mailing_address": profile.mailing_address,
             "requires_parental_consent": profile.requires_parental_consent(),
+            "account_privacy": UserPreference.get_value(user, 'account_privacy'),
```

Comment:
Make sure that all the documentation is updated. This includes the API documentation (https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/user_api/accounts/views.py#L36) and the wiki spec-- https://openedx.atlassian.net/wiki/display/TNL/User+API.

Also, I believe that this API is documented in our edx docs. Please check with @catong or @mhoeber about this.